### DISCUSSION TITLE
Exporting plots as PDFs

### DISCUSSION THREAD

**Discussion by burnt35** [ImmuneDynamics/Spectre#86] [Q&A] [answered]
Hi Tomas,
I am loving the work you've put into spectre and for making it so accessible! Thanks!

I was wondering if there was a way to export plots you make as PDFs. I see you can do this for the make.pheatmap function, but I can't work it out with make.multi.plot or make.colour.plot.

Thanks!
Tom

**Comment by tomashhurst** [comment, marked as answer]
Hi @burnt35 , indeed there is a reasonably quick way to facilitate this -- if you use the 'filename' argument, you can use '.pdf' at the end (e.g. filename = 'test.pdf') and that should take care of it.

**Reply by burnt35** [reply]
Brilliant, thank you!
Tom

**Reply by burnt35** [reply]
Seems to work fine for the make.colour.plot but not make.multi.plot.
Not a major, just to let you know. Thanks!

**Reply by tomashhurst** [reply]
Ah yes, a didn't put a 'filename' argument in multiplot. It's on the list for the next release!
